packages/core/src/server/api.ts

```diff
@@ -1,11 +1,13 @@
 import { MultihashDigest } from 'multiformats/hashes/digest'
 import { UnknownLink, Link } from 'multiformats/link'
+import { AnyAssertCap } from './service/api'
 
 export interface Claim {
   claim: Link
   bytes: Uint8Array
   content: MultihashDigest
-  expiration?: number
+  expiration?: number,
+  value: AnyAssertCap
```

Comment:
passing the full capability object through to the claims store allows us to inspect the capability name, e.g. `assert/equals` and special case it, and we also get the `nb` property on the capability, so we can extract the `equals` cid.

Reply:
also it has a pleasing symmetry with the mutliformats block api which has the `bytes` and the decoded `value` of a block.